src/serializer/ResidualHeapVisitor.js

```diff
@@ -1333,36 +1318,4 @@ export class ResidualHeapVisitor {
       }
     }
   }
-
-  _visitReactLibrary(someReactElement: ObjectValue) {
-    // find and visit the React library
-    let reactLibraryObject = this.realm.fbLibraries.react;
-    if (this.realm.react.output === "jsx") {
-      // React might not be defined in scope, i.e. another library is using JSX
-      // we don't throw an error as we should support JSX stand-alone
-      if (reactLibraryObject !== undefined) {
-        this.visitValue(reactLibraryObject);
-      }
-    } else if (this.realm.react.output === "create-element") {
-      let logError = () => {
-        this.logger.logError(
```

Comment:
This error wouldn’t be recoverable since we would get an `invariant` exception in our serializer. So I updated the code to throw a `FatalError` instead.